cmd/snaptel/task.go

```diff
@@ -385,10 +385,15 @@ func createTaskUsingTaskManifest(ctx *cli.Context) error {
 	r := pClient.CreateTask(t.Schedule, t.Workflow, t.Name, t.Deadline, !ctx.IsSet("no-start"), t.MaxFailures)
 
 	if r.Err != nil {
-		errors := strings.Split(r.Err.Error(), " -- ")
-		errString := "Error creating task:"
-		for _, err := range errors {
-			errString += fmt.Sprintf("%v\n", err)
+		errString := "Error creating task: " + fmt.Sprintf("%v\n", r.Err.Error())
+
+		if r.Fields["body"] != nil {
+			fieldsInsideFieldsBody, _ := r.Fields["body"].(map[string]interface{})
+			fieldsInsidefields, _ := fieldsInsideFieldsBody["fields"].(map[string]interface{})
```

Comment:
This structure represents content of fields. For example with error `ErrPluginIncompatibleWithScheduleType` this structure could be like this one:
`map[plugin_name:psutil plugin_type:collector schedule_type:*schedule.StreamingSchedule]`